assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -173,4 +173,19 @@ export const isMobile = {
 // works for SCREAMING_SNAKE_CASE, snake_case or cRazY_SNAKE_case
 export const snakeToCamel = (string: string) => string.toLowerCase().replace(/_[a-z]/g, match => match[1].toUpperCase());
 
-export const youtubeRegexp = /(?:youtube\.com\/(?:[^/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?/ ]{11})/i;
\ No newline at end of file
+export const youtubeRegexp = /(?:youtube\.com\/(?:[^/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?/ ]{11})/i;
+
+export const getCookieItem = (sKey: string) => {
+  if (!sKey) {
+    return null;
+  }
+  return (
+    decodeURIComponent(
+      document.cookie.replace(
+        // eslint-disable-next-line no-useless-escape
+        new RegExp(`(?:(?:^|.*;)\\s*${encodeURIComponent(sKey).replace(/[\-\.\+\*]/g, '\\$&')}\\s*\\=\\s*([^;]*).*$)|^.*$`),
```

Comment:
Currently our cookies are pretty small. So, creating this `RegExp` obj each time this method has limited cost, especially since it's being called on the critical path of the front-end. I would be curious to know what the cost/benefit would be of caching this result, perhaps using:
https://www.npmjs.com/package/regex-cache

Reply:
This is micro benchmarking here. This won't change anything in time to render.

Reply:
I don't expect much change for the moment. But, this would be a place for optimization if/when our cookies grow.